server/db/db.js

```diff
@@ -89,6 +91,15 @@ async function beginTransaction(retryCount = 0) {
   return pgClient
 }
 
+async function beginTransaction() {
+  try {
+    const pgClient = await runBeginTransactionWithRetries(0)
+    return pgClient
```

Comment:
Thanks Theresa, I originally tried to have it as one line but our ESLint wouldn't allow it, calling it "redundant". But I looked into the ESLint error documentation: https://eslint.org/docs/latest/rules/no-return-await and it turns out this rule was deprecated, so that's why I had to implement it this way.

Reply:
Haha, OK. Thanks for trying! All good :smile: